README.md

```diff
@@ -101,14 +101,409 @@ download or scan any previously scanned sources, the Scanner can be configured (
+        name: "builtin-modules"
+        version: "1.1.1"
+        dependencies: []
+        errors: []
+    # ...
+# Detailed metadata about each package from the dependency trees.
+packages:
+- id:
+    provider: "NPM"
+    namespace: ""
+    name: "abbrev"
+    version: "1.0.9"
+  declared_licenses:
+  - "ISC"
+  description: "Like ruby's abbrev module, but in js"
+  homepage_url: "https://github.com/isaacs/abbrev-js#readme"
+  binary_artifact:
+    url: "https://registry.npmjs.org/abbrev/-/abbrev-1.0.9.tgz"
+    hash: "91b4792588a7738c25f35dd6f63752a2f8776135"
+    hash_algorithm: "SHA-1"
+  source_artifact:
+    url: ""
+    hash: ""
+    hash_algorithm: ""
+  vcs:
+    type: "git"
+    url: "git+ssh://git@github.com/isaacs/abbrev-js.git"
+    revision: "c386cd9dbb1d8d7581718c54d4ba944cc9298d6f"
+    path: ""
+  vcs_processed:
+    type: "Git"
+    url: "ssh://git@github.com/isaacs/abbrev-js.git"
+    revision: "c386cd9dbb1d8d7581718c54d4ba944cc9298d6f"
+    path: ""
+# ...
+#  Finally a list of errors that happened during dependency analysis. Fortunately empty in this case.
+errors: []
+```
+
+If you try the commands above with a different NPM package that does not have a
+[package-lock.json](https://docs.npmjs.com/files/package-locks) (or `npm-shrinkwrap.json` or `yarn.lock`) the analyzer
+will terminate with an error message like this: 
+
+```
+ERROR - Analysis for these projects did not complete successfully:
+[npm-project-path]/package.json
+```
+
+This means that there have been issues with the dependency resolution of these packages. The reasons for these errors
+can be found in the log output of the `analyzer` or in the results file:
+
+```
+Resolving NPM dependencies for '[npm-project-path]/package.json'...
+17:11:16.683 ERROR - Resolving dependencies for 'package.json' failed with: No lockfile found in [npm-project-path], dependency versions are unstable.
+```
+
+This happens because without a [lockfile](https://docs.npmjs.com/files/package-locks) the versions of transitive
+dependencies could change at any time. Therefore ORT checks for the presence of a lockfile to generate reliable results.
+This check can be disabled with the `--allow-dynamic-versions` option.
+
+**5. Run the scanner**
+
+To scan the source code of `mime-types` and its dependencies the source code of `mime-types` and all its dependencies
+needs to be downloaded. The `downloader` tool could be used for this, but it is also integrated in the `scanner` tool,
+so the scanner will automatically download the source code if the required VCS metadata could be obtained.
+
+ORT is designed to integrate lots of different scanners and is not limited to license scanners, technically any tool
+that explores the source code of a software package could be integrated. Currently these scanners are available:
+
+* [BoyterLc](https://github.com/boyter/lc)
+* [Licensee](https://github.com/benbalter/licensee)
+* [ScanCode](https://github.com/nexB/scancode-toolkit)
```

Comment:
Again, this duplicates the existing list of "Supported license scanners". Is it worth repeating, or should be simply say we use the default scanner of ScanCode for this tutorial.